### PR TITLE
Create a Cancel button to the opposite of the "Create Resource" button.

### PR DESCRIPTION
**Pull request by engineSound**
### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [x] Passing Jenkins Build

- [x] Peer Code review and approval

### Positive Test Case
1. Initiate a Create a new resource action from "My Resources".
2. In the page where I create a new resource, scroll to the bottom of the page. On the opposite of the "Create Resource" button, find the "Cancel" button.
3. Click the "Cancel" button, the browser goes back to the "My Resources" page.

### CODE REVIEW

**Comment by hydrobuild** [comment]
Build finished.

File: theme/static/js/create-resource.js [hunk 1]
```diff
@@ -97,6 +97,10 @@ $(document).ready(function () {
                 }
             });
 
+            $(".btn-cancel-create-resource").click(function () {
+                history.back();
```
Comment (aphelionz) [inline theme/static/js/create-resource.js]: Let's make this explicitly go to the my resources page via `window.location.` `history.back` can sometimes be unpredictable.

Comment (engineSound) [reply theme/static/js/create-resource.js]: "Create Resource page" might be invoked from another location when design change in the future. Using "history.back" can prevent we revisit the code again.

Comment (aphelionz) [reply theme/static/js/create-resource.js]: Ok, fair point. Ship it

**Pull request merged**